### PR TITLE
GDPR updates

### PR DESCRIPTION
**Pull request by chomipi88**

### CODE REVIEW

File: astro/security.md [hunk 1]
```diff
@@ -38,7 +38,9 @@ Astronomer is a global remote company first. We have some small offices in the U
 
 Astro is AICPA SOC 2 Type I certified with respect to the security, availability, and confidentiality Trust Service Categories. If you are interested in obtaining (NDA required) our SOC 2 Report and Penetration Test report, contact [sales@astronomer.io](https://www.notion.so/Sales-Operations-36c4d78df58747a2815449b7acbe97bd)
 
-Astronomer is committed to pursuing SOC 2 Type II, GDPR, and HIPAA compliance in 2022.
+Astronomer is also GDPR-compliant as an organization, and the Astro platform is GDPR-ready. Astronomer now offers a Data Processing Agreement (DPA), which satisfies the requirements the GDPR imposes on data controllers with respect to data processors.
```
Comment (astrorfox) [inline astro/security.md:41]:
```suggestion
Astronomer is also GDPR-compliant as an organization, and the Astro platform is GDPR-ready. Astronomer offers a Data Processing Agreement (DPA), which satisfies the requirements the GDPR imposes on data controllers with respect to data processors.
```

**Review by astrorfox — COMMENTED** [review]
@chomipi88 can we remove the reference to time (i.e. "now")?

**Comment by chomipi88** [comment]
> @chomipi88 can we remove the reference to time (i.e. "now")?

done

**Review by paolaperaza — APPROVED** [review]

**Comment by paolaperaza** [comment]
Approved! The wording on GDRPR ready and "Astro platform" feels a little off, but I'll defer to you two on language and we can tweak after - @astrorfox @chomipi88 Feel free to merge whenever you guys are ready

**Pull request merged**